### ISSUE TITLE
Content of toggle headers incorrectly indented as code block

### ISSUE THREAD

**Issue by robertandrews** [anhi/obsidian-notion-connector#2]
Looks like content of a toggle header is getting tab-indented, 

Compare the Notion page on the left with the Obsidian grab on the right. Obsidian as seen is the Default theme in 1.0.0 - the theme font is not a monospaced font, nor have I overridden to make it such...

<img width="1068" alt="Screenshot 2022-10-24 at 15 41 36" src="https://user-images.githubusercontent.com/3158787/197553881-fae37203-d982-4ef5-a47c-e1b60fae6054.png">

Inspecting Markdown shows many tab-indents...

<img width="596" alt="Screenshot 2022-10-24 at 15 44 42" src="https://user-images.githubusercontent.com/3158787/197554669-4a4968b2-d4c8-4e4d-9107-03f0736595ab.png">

[In Markdown, tab-indents denote a code block](https://www.markdownguide.org/basic-syntax/#code-blocks). Thus, what should be plain text is being written as code block.

The example also seems to show excess line breaks?

---

I believe the content of a toggle header should get rendered as follows...

* Retain the header size (eg. toggle header 3 becomes `###`).
* Text content becomes plain text content.

In other words, I don't think you should adopt the `<details>` and `<summary>` HTML markup you have used for toggles generally...
